### ISSUE TITLE
PostGIS support

### ISSUE THREAD

**Issue by idcc-cursor** [opengisch/qfieldsync#393]
Hello Qfield team - 1st. of all to you, qfield is a great piece of software.
I regoniced that PostGIS is not much supported by qfieldsync pugin....

I use the latest qfieldsysnc experimental witg qgis 3.16 LTS. Layers and qgis project file is stored in PostGIS.
When I export, qgis freeze after the export with a "Enter Credentials" window.

Any ideas ?

**Comment by suricactus** [comment]
@idcc-cursor can you please share details how you store your credentials and what action did you configure your layers with. Also share your QFieldSync verion.

**Comment by idcc-cursor** [comment]
Hello,
1) I configured my PostGIS connection in Basic mode. Credential stored as plain text in project file.
2) Qfieldsync Version: 4.0.0-BETA18 - QGIS 3.16.16 LTR
3) I stored QGIS Project file into PostGIS
4) I configured some vector layers as cable export offline editing, google maps orientaion as direct access. Project and layer projektion is EPSG:31255
5) I exported project
7) After export QGIS freeze with "Enter Credentials" window

I saw that ther is a new QGIS LTR available and a qfiledsync BETA19 too. I will update as soon as possible and test again. Hope that information helps to make this great piece of software stable. Thank you !! Best regards Andreas
